### ISSUE TITLE
Next scenario after Operation Checkmate in US campaign 

### ISSUE THREAD

**Issue by vbaronpg** [nicupavel/openpanzer#259]
I finished the Operation Checkmate with a Brilliant victory and the next scenario that appears is Prokhorovka with Russians. Running the app on an iPAD with iOS 13.3.3 and v3.2.6.  Anyone else see this issue?

Thanks 
Vin

**Comment by salexgit** [comment]
Hi

Can you upload here the saved game? The turn before you capture the last victory hex.

**Comment by vbaronpg** [comment]
Idiot that I am, I accidentally deleted it.  I will start again and maybe
it will re-occur.

Vin

**Comment by vbaronpg** [comment]
Ok, I got it to happen on both my iPAD (via the app) and PC (via PanzerMarshal.com)

I changed the extension from .json to .txt to attach it here.

[(Campaign) Operation Checkmate Turn 11 Thu Mar 26 2020 7_00_30 PM.txt](https://github.com/nicupavel/openpanzer/files/4390451/Campaign.Operation.Checkmate.Turn.11.Thu.Mar.26.2020.7_00_30.PM.txt)

**Comment by vbaronpg** [comment]
Oops, wrong seletction

**Comment by salexgit** [comment]
Hi

@vbaronpg: I tried to load your saved game, but it does not load at all (neither 3.2.6 nor 3.2.7 works). Which game version do you use?

**Comment by vbaronpg** [comment]
I transferred it via the Cloud to Firefox at PanzerMarshall.com using
3.2.7.  I attached the .json file. Here is the Cloud Save file:

3ae33cbbc6f96c44fcab44a8b3108b93

I get the same error on both my iPAD (v3.2.6)  and my PC using the above
method with the same results.

Thanks,
Vin

**Comment by salexgit** [comment]
Hi

@vbaronpg: I was able to load your game in 3.2.7, but it is "Operation Dragoon", not "Operation Checkmate". Or am I getting something wrong?

**Comment by vbaronpg** [comment]
I think I overwrote it with another game. Try again please.

3ae33cbbc6f96c44fcab44a8b3108b93

**Comment by salexgit** [comment]
Hi

I was able to load the game, but both in v3.2.6 and v.3.2.7 it works correctly for me - the next scenario was "Prokhorovka". I tried both local and online versions.

Regards, Alex

**Comment by vbaronpg** [comment]
I was playing the US campaign, it goes to Russia after that?  I thought
that was an error, sorry if it is not.

**Comment by salexgit** [comment]
Hi

Ok, now I got it. Funny and interestingly enough, this US campaign has several Soviet/German scenarios, such as Prokhorovka, Leningrad, and Kanev. I have no clue why they were added there because it does not make any historical sense. On the other hand, the whole campaign is pseudo-historical because in 1945, after defeating Germany, you are supposed to fight against Soviet Union.
I will check with the main developer why to do. It is a bit strange one.

Regards, Alex
